todo_app/templates/index.html

```diff
@@ -7,11 +7,21 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <div>
+    <form action="/" method="post">
+      <input type="text" name="additem">
+      <button type="submit">submit</button>
+    </form>
+ </div>
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{item.id}} - {{item.status}}</li>
```

Comment:
It would be nice to also include the todo's title here - knowing that todo id 4 isn't started yet isn't very useful if we can't remember what that was meant to be!